testsuite/features/secondary/srv_task_status_engine.feature

```diff
@@ -0,0 +1,46 @@
+# Copyright (c) 2022 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+Feature: Task Engine Status
+
+  Scenario: Login as admin
+    Given I am authorized for the "Admin" section
+    
+  Scenario: Check if the Task Engine Status page exists
+    When I follow the left menu "Admin > Task Engine Status > Last Execution Times"
+    Then I should see a "Task Engine Status" text
+    And I should see a "The following is a status report for the various tasks run by the Uyuni task engine:" text
+    And I should see a "Runtime Status" text
+    And I should see a "Last Execution Times" link in the left menu
+    And I should see a "Runtime Status" link in the left menu
+
+  Scenario: Check if the Runtime Status Page exists
+    When I follow the left menu "Admin > Task Engine Status > Runtime Status"
+    Then I should see a "Task Engine Status" text
+    And I should see a "Last Execution Times" text
+    And I should see a "The server is running or has finished executing the following tasks during the latest 5 minutes." text
+    And I should see a "Last Execution Times" link in the left menu
+    And I should see a "Runtime Status" link in the left menu
+  
+  Scenario: Run a remote command on the server to have and check if it shows up on Last Execution Times page
```

Comment:
>   Scenario: Run a remote command on the server to have...

I still do not know what you mean with that part of this sentence.

Reply:
Me neither, thought I corrected that grammar mistake. Fixing it now :)